file_io.cpp

```diff
@@ -55,7 +57,7 @@ int FileOpenEx(fileTYPE *file, const char *name, int mode, char mute)
 	}
 	else
 	{
-		sprintf(full_path, name);

```

Comment:
It's generally considered a security issue to not use a hardcoded format string since if you can manipulate the input string you can get sprintf to dump data from the current context.  For this application I guess it's not a big issue though.

Reply:
MiSTer isn't compiled with security thoughts :)